insets/src/main/java/com/google/accompanist/insets/WindowInsets.kt

```diff
@@ -509,4 +542,11 @@ annotation class ExperimentalAnimatedInsets
 /**
  * Composition local containing the current [WindowInsets].
  */
+@Deprecated(
+"""
+accompanist/insets is deprecated.
+The androidx.compose equivalent of LocalWindowInsets is the extensions on WindowInsets.
+For more migration information, please visit https://google.github.io/accompanist/insets/#migration
```

Comment:
Do we need to publish those docs first before merging this?

Reply:
I'm not entirely sure what the doc publishing process is, will those go out immediately after publishing?

If so, the updated `docs/insets.md` in this PR should make this link real after the merge.

Reply:
Oh sorry! I completely missed that you had changed it because GitHub had collapsed it